easybuild/easyconfigs/k/KNIME/KNIME-3.6.2.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'KNIME'
+version = '3.6.2'
+
+homepage = 'https://www.knime.com/'
+
+description = """ 
+KNIME Analytics Platform is the open source software
+for creating data science applications and services.
+KNIME stands for KoNstanz Information MinEr. 
+"""
+
+source_urls = ['https://download.knime.org/analytics-platform/linux/']
+sources = ['%(namelower)s_%(version)s.linux.gtk.x86_64.tar.gz']
+checksums = ['78a44bb5d3d86ed7251181d2d25d3d93c297b652027c6884f66ba45e49025119']
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+dependencies = [('Java', '1.8.0_172')]
```

Comment:
Please use the `Java/1.8` wrapper module here rather than fixing to a specific `Java` version:

```python
dependencies = [('Java', '1.8')]
```

(see also https://easybuild.readthedocs.io/en/latest/Wrapping_dependencies.html)